Bluepill-cli/Bluepill-cli/Simulator/BPSimulator.m

```diff
@@ -75,29 +75,33 @@ - (NSMutableDictionary *)createSimulatorAndInstallAppWithBundles:(NSArray<BPXCTe
     return testHostForSimUDID;
 }
 
-- (NSString *)installApplicationWithHost:(NSString *)testHost withError:(NSError *)error {
-    SimServiceContext *sc = [SimServiceContext sharedServiceContextForDeveloperDir:self.config.xcodePath error:&error];
+- (NSString *)getErrorDescription:(NSError *__autoreleasing *)error {
+    return error != nil ? [*error localizedDescription] : nil;
+}
+
+- (NSString *)installApplicationWithHost:(NSString *)testHost withError:(NSError *__autoreleasing *)error {
```

Comment:
Change `(NSError *__autoreleasing *)error` to `(NSError *__autoreleasing *)errorPtr`, otherwise the code below is hard to understand.. like SimDeviceSet *deviceSet = [sc defaultDeviceSetWithError:error];

Reply:
It seems to be the convention in Apple APIs 🤔 ..

Reply:
You mean using `error`... I think they just lean on the type checker... I'm ok either way.

Reply:
Did a couple updates on my comment.. 

We should be clear if it is ** we name it `errPtr`, if it is * call it `error`

Reply:
It would be great if @jerrymarino could do a find & replace for the whole code base :)

Reply:
+1 for calling it `errorPtr` or something different than error. I left this as error since it's what the rest of the codebase uses 😉 .

It's definitely going to be some more effort to the rename, I'll try to get to it later